bin/interpreter.sh

```diff
@@ -128,6 +131,8 @@ if [[ "${INTERPRETER_ID}" == "spark" ]]; then
   fi
 fi
 
+addJarInDir "${LOCAL_REPO_DIR}"
```

Comment:
local repo would be [.m2](https://maven.apache.org/guides/mini/guide-configuring-maven.html)? don't we need to recursively add all jars in subdirectory then?